### PR TITLE
TC-739 info button

### PR DESCRIPTION
**Pull request by kscoulter**
Add info button to primary search bar. Toggle text on click.

Screen Shot 2021-04-19 at 3 12 14 PM

Screen Shot 2021-04-19 at 3 12 08 PM

Screen Shot 2021-04-19 at 3 12 00 PM

### CODE REVIEW

File: src/client/components/atoms/button/button.scss [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 .primary:not([disabled]) {
   background-color: $sba-red;
-  color: white;
+  color: $sba-white;
```
Comment (peggles2) [inline src/client/components/atoms/button/button.scss:43]: nice job adding a variable

File: src/client/components/organisms/primary-search-bar/primary-search-bar.scss [hunk 2]
```diff
@@ -1,20 +1,3 @@
...
-//     @include grid-column(12);
-//     margin-top: 20px;
-//     margin-bottom: 30px;
-//   }
-//   @include grid-media($sba-grid-large) {
-//     @include grid-column(12);
-//     //    @include grid-push(2);
-//   }
-// }
-
```
Comment (peggles2) [inline src/client/components/organisms/primary-search-bar/primary-search-bar.scss:17]: nice job cleaning up some code.

File: test/jest/client/components/organisms/primary-search-bar/primary-search-bar.test.js [hunk 3]
```diff
@@ -87,14 +87,35 @@ describe('Primary Search Bar', () => {
...
     const button = searchBar.find('Button')
     expect(button.props().disabled).toBeTruthy()
   })
+
   it('will NOT disable the button if the button is set to active', () => {
     const searchBar = mount(<PrimarySearchBar searchButtonText="foo" buttonActive={true} />)
     const button = searchBar.find('Button')
     expect(button.props().disabled).not.toBeTruthy()
   })
+
```
Comment (peggles2) [inline test/jest/client/components/organisms/primary-search-bar/primary-search-bar.test.js:102]: Can you also write a test of what the expected behavior should be during initial page load? So the info text does not display initially.

**Review by peggles2 — CHANGES_REQUESTED** [review]
Looks good just a few comments.

File: src/client/components/atoms/button/button.scss [hunk 4]
```diff
@@ -100,6 +100,21 @@
...
+  &:hover {
+    outline: 2px dotted #c5c5c5;
+    outline-offset: 3px;
+  }
+
+  &:global(.blue) {
+    background-color: $secondary-blue;
+  }
+}
+
```
[the remark this answers is not in the document]
Comment (kscoulter) [reply src/client/components/atoms/button/button.scss:117]: Not sure what you mean by this.

[the remark this answers is not in the document]
Comment (peggles2) [reply src/client/components/atoms/button/button.scss:117]: placing it in variables.scss

[the remark this answers is not in the document]
Comment (kscoulter) [reply src/client/components/atoms/button/button.scss:117]: secondary blue is already a variable

**Review by peggles2 — APPROVED** [review]
LGTM 👍

**Review by peggles2 — APPROVED** [review]
LGTM 👍

**Pull request merged**